pithos/pithos.py

```diff
@@ -328,6 +419,22 @@ def init_actions(self, app):
         app.add_accelerator('<Primary>d', 'win.bookmark', None)
         action.connect('activate', self.bookmark_song)
 
+    def style_context_change_handler(self, style_context):
+        #We only care if the window or icon theme has changed.
+        current_theme = Gtk.Settings.get_default()
+        current_window_theme_name = current_theme.props.gtk_theme_name
+        current_icon_theme_name = current_theme.props.gtk_icon_theme_name
+        if (self.current_window_theme_name != current_window_theme_name
+            or self.current_icon_theme_name != current_icon_theme_name):
+            self.current_window_theme_name = current_window_theme_name
+            self.current_icon_theme_name = current_icon_theme_name
+            #Update rating icons and background, and generic cover icon and background.
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->